### PR TITLE
Postfix looks for vdomains alias at LDAP

### PR DESCRIPTION
**Pull request by figarocorso**

### CODE REVIEW

**Comment by blaxter** [comment]
I don't know the reason (because the commit message is not very explicit) but in zentyal there is another query https://github.com/Zentyal/zentyal/commit/ef83f67f610414b41e35436853c1500850586df1 for this

File: modules/zentyalcloud/templates/mta/domain_aliases.cf.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+server_host = <%= @ldap_host %>
+version = 3
+search_base =  <%= @ldap_basedn %>
+query_filter = (& (mail=@%s)(objectClass=CourierMailAlias))
```
Comment (exekias) [inline modules/zentyalcloud/templates/mta/domain_aliases.cf.erb]: the space betwen & and ( sounds bad to me, does it work?

File: modules/zentyalcloud/templates/mta/main.cf.erb [hunk 2]
```diff
@@ -56,5 +56,5 @@ dovecot_destination_recipient_limit = 1
 virtual_transport = dovecot
 virtual_mailbox_domains = ldap:/etc/postfix/vdomains.cf
 virtual_alias_domains = $virtual_alias_maps
-virtual_alias_maps = ldap:/etc/postfix/user_aliases.cf ldap:/etc/postfix/group_aliases.cf
+virtual_alias_maps = ldap:/etc/postfix/user_aliases.cf ldap:/etc/postfix/group_aliases.cf ldap:/etc/postfix/domain_aliases.cf
```
Comment (exekias) [inline modules/zentyalcloud/templates/mta/main.cf.erb]: are you writing that file /etc/postfix/domain_aliases.cf ?

Comment (figarocorso) [inline modules/zentyalcloud/templates/mta/main.cf.erb]: Hahaha, problems. You are totally right, missed that.
![Image]
(http://media.giphy.com/media/c1inrwnU8rjq/giphy.gif)

File: CHANGES.md [hunk 3]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The descriptions should be useful and understandable for end users of ZINC.
 Unreleased changes refer to our current [master branch](https://github.com/Zentyal/zinc-puppet/).
 
+## [Unreleased]
+
+### Added
+* MTA accepts mails to vdomain aliases
```
Comment (exekias) [inline CHANGES.md]: I would say something more like: Support for mail domain alias

**Comment by figarocorso** [comment]
@blaxter That commit is old. But I've done some of the changes there. Thank you.

File: modules/zentyalcloud/templates/mta/main.cf.erb (hunk 2 above)
Comment (exekias) [inline modules/zentyalcloud/templates/mta/main.cf.erb]: I love arturito

**Pull request merged**